microshift-okd/src/build.sh

```diff
@@ -0,0 +1,41 @@
+#/bin/bash
+
+
+# replace microshift assets to Upsteam from OKD
+okd_url=quay.io/okd/scos-release
+okd_releaseTag=4.17.0-0.okd-scos-2024-08-21-100712
+
+oc adm release info ${okd_url}:${okd_releaseTag} >all_images
+
+for op in $(cat assets/release/release-x86_64.json  | jq -e -r  '.images | keys []') 
+do
+    image=$(oc adm release info --image-for=${op} ${okd_url}:${okd_releaseTag} || true) 
+    if [ -n "${image}" ] ; then
+        echo "${op} ${image}"
+        jq --arg a "${op}" --arg b "${image}"  '.images[$a] = $b' assets/release/release-x86_64.json >/tmp/release-x86_64.json.tmp
+
+
+        # delete openssl image from assets - just to verify if we still need it,since it doesnt referenced anywhere
+        jq '. | del (.images["openssl"])'  assets/release/release-x86_64.json >/tmp/release-x86_64.json.tmp2
+
+        mv /tmp/release-x86_64.json.tmp2 assets/release/release-x86_64.json
+    fi
+done
+
+
+sudo podman stop microshift-okd
+rm -rf microshift-okd/src/rpmbuild
+rm -rf _output/rpmbuild
+make build 
+make rpm
+createrepo _output/rpmbuild/
+cp -rf _output/rpmbuild microshift-okd/src/
```

Comment:
It should be done the other way around. We should not copy binaries into the source directory.
Instead, copy `okd/src` to `_output/okd/...` and run podman builds from there.

Reply:
yes, i had to do it because of podman build context.
 see the other approach using multi-build.